src/tools/run-make-support/src/fs.rs

```diff
@@ -176,3 +176,13 @@ pub fn set_permissions<P: AsRef<Path>>(path: P, perm: std::fs::Permissions) {
         path.as_ref().display()
     ));
 }
+
+/// A function which prints all file names in the directory `dir` similarly to Unix's `ls`.
+/// Useful for debugging.
+#[track_caller]
+pub fn list_files<P: AsRef<Path>>(dir: P) {
+    let paths = read_dir(dir);
+    for path in paths {
+        println!("file name: {}", path.unwrap().path().display())
+    }
+}
```

Comment:
Discussion: would this be generally more useful if it just *returned* the dir entries (`-> Vec<PathBuf>`), then the callsite in tests can just do

```rs
eprintln!("{:#?}", shallow_find_dir_entries(some_dir));
```

(Bikeshedding: dir entries might be more accurate since it's not just "files" files, but also directories, and this operation is a shallow find).